Doc/library/imaplib.rst

```diff
@@ -582,6 +585,12 @@ An :class:`IMAP4` instance has the following methods:
 
    Unsubscribe from old mailbox.
 
+.. method:: IMAP4.unselect()
+
+   Close the current mailbox without removing messages from
+   the currently selected mailbox.
```

Comment:
The RFC definition looks more precise:

      The UNSELECT command frees server's resources associated with the
      selected mailbox and returns the server to the authenticated
      state.  This command performs the same actions as CLOSE, except
      that no messages are permanently removed from the currently
      selected mailbox.

https://tools.ietf.org/html/rfc3691#section-2

With your definition, I didn't understand that the connection remains valid but a new authentication is needed. But I'm not used to the IMAP protocol.